public/index.html

```diff
@@ -48,18 +50,29 @@
         </div>
       </div>
       <div class="sidebar">
-        <div id="editor">var playerElement = document.getElementById("player-wrapper");
+        <div id="editor">
+var clapprDemo = window.clapprDemo = {}
+
+var playerElement = document.getElementById("player-wrapper");
 
 var player = new Clappr.Player({
-  source: 'http://clappr.io/highline.mp4',
+  source: 'https://video-dev.github.io/streams/x36xhzz/x36xhzz.m3u8',
```

Comment:
this is my wip, we can have the mp4 as well in there :)

But it would be cool to show a bit more in the demo the HLS and DASH integrations of Clappr, also to keep making sure all of this stuff integrates well with all the plugins (like level-switching, captions etc)